Frends.AzureDataLake.UploadFiles/Frends.AzureDataLake.UploadFiles/Definitions/Result.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+
+namespace Frends.AzureDataLake.UploadFiles.Definitions;
+
+/// <summary>
+/// Task's result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Operation complete.
```

Comment:
Lets do something along the lines of "True if operation was successful, false otherwise. Note that if a file already exists and the overwriting is turned off, the task result will be successful (true). Setting ThrowErrorOnFailure to true will make that task throw an exception if an error occurs instead of setting Success to false."